Found some more information, looks like I just don't have the most recent OMERO.marshal available locally, right (`"No decoder found for type: http://www.openmicroscopy.org/Schemas/OME/2016-06#Screen"`)?

```
__________________ TestContainers.test_container_crud[Screen] __________________
self = <test_api_containers.TestContainers object at 0x109c56c50>
dtype = 'Screen'
    @pytest.mark.parametrize("dtype", ['Project', 'Dataset', 'Screen'])
    def test_container_crud(self, dtype):
        """
            Test create, read, update and delete of Containers.
            Create with POST to /save
            Read with GET of /m/dtype/:id/
            Update with PUT to /m/dtype/:id/
            Delete with DELETE to /m/dtype/:id/
            """
        django_client = self.django_root_client
        group = self.ctx.groupId
        version = settings.API_VERSIONS[-1]
        # Need to get the Schema url to create @type
        base_url = reverse('api_base', kwargs={'api_version': version})
        rsp = _get_response_json(django_client, base_url, {})
        schema_url = rsp['schema_url']
        # specify group via query params
        save_url = "%s?group=%s" % (rsp['save_url'], group)
        project_name = 'test_container_create_read'
        payload = {'Name': project_name,
                   '@type': '%s#%s' % (schema_url, dtype)}
        rsp = _csrf_post_json(django_client, save_url, payload,
>                             status_code=201)
test/integration/test_api_containers.py:112: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
django_client = <django.test.client.Client object at 0x10a1a6bd0>
request_url = 'http://testserver/api/v0.1/m/save/?group=5'
data = {'@type': 'http://www.openmicroscopy.org/Schemas/OME/2016-06#Screen', 'Name': 'test_container_create_read'}
status_code = 201, content_type = 'application/json'
    def _csrf_post_json(django_client, request_url, data,
                        status_code=200, content_type='application/json'):
        csrf_token = django_client.cookies['csrftoken'].value
        extra = {'HTTP_X_CSRFTOKEN': csrf_token}
        rsp = django_client.post(request_url, json.dumps(data),
                                 status_code=status_code,
                                 content_type=content_type,
                                 **extra)
        print rsp
>       assert rsp.status_code == status_code
E       assert 400 == 201
E        +  where 400 = 400
E        +    where 400 = <django.http.response.JsonResponse object at 0x109c48d10>.status_code
../target/lib/python/omeroweb/testlib/__init__.py:149: AssertionError
----------------------------- Captured stdout call -----------------------------
Vary: Cookie
Content-Type: application/json
{"message": "No decoder found for type: http://www.openmicroscopy.org/Schemas/OME/2016-06#Screen"}
_____________________________ TestCsrf.test_script _____________________________
self = <test_csrf.TestCsrf object at 0x10a350590>
    def test_script(self):
        img = self.create_test_image(session=self.sf)
        script_path = "omero/export_scripts/Batch_Image_Export.py"
        script = self.sf.getScriptService().getScriptID(script_path)
>       request_url = reverse('script_run', args=[script])
test/integration/test_csrf.py:274: 
```

--- a/components/tools/OmeroWeb/requirements-common.txt
+++ b/components/tools/OmeroWeb/requirements-common.txt
@@ -6,4 +6,8 @@
 
 Django>=1.8,<1.9
 django-pipeline==1.3.20
-omero-marshal==0.4.1
+
+# Until Screens are supported in release, we need to use master branch
+# See https://github.com/openmicroscopy/openmicroscopy/pull/5006
+# omero-marshal==0.4.1
+git+git://github.com/openmicroscopy/omero-marshal.git@master#egg=omero-marshal

--- a/components/tools/OmeroWeb/test/integration/test_api_containers.py
+++ b/components/tools/OmeroWeb/test/integration/test_api_containers.py
@@ -28,21 +28,6 @@ from omero.gateway import BlitzGateway
 from omero_marshal import get_encoder
 
 
-def get_update_service(user):
-    """Get the update_service for the given user's client."""
-    return user[0].getSession().getUpdateService()
-
-
-def get_connection(user, group_id=None):
-    """Get a BlitzGateway connection for the given user's client."""
-    connection = BlitzGateway(client_obj=user[0])
-    # Refresh the session context
-    connection.getEventContext()
-    if group_id is not None:
-        connection.SERVICE_OPTS.setOmeroGroup(group_id)
-    return connection
-
-
 def marshal_objects(objects):
     """Marshal objects using omero_marshal."""
     expected = []
@@ -77,14 +62,6 @@ def assert_objects(conn, json_objects, omero_ids_objects, dtype="Project",
 
 class TestContainers(IWebTest):
     """Tests querying & editing Datasets, Screens etc."""
-
-    # Create users in the read-only group
-    @pytest.fixture()
-    def user1(self):
-        """Return a new user in the group1 group and also add to group2."""
-        group = self.new_group(perms='rwra--')
-        user = self.new_client_and_user(group=group)
-        return user
 
     @pytest.mark.parametrize("dtype", ['Project', 'Dataset', 'Screen'])
     def test_container_crud(self, dtype):

--- a/components/tools/OmeroWeb/test/integration/test_api_containers.py
+++ b/components/tools/OmeroWeb/test/integration/test_api_containers.py
@@ -93,7 +93,6 @@ class TestContainers(IWebTest):
 
         # Read Object
         object_url = "%sm/%ss/%s/" % (base_url, dtype.lower(), object_id)
-        print object_url
         object_json = _get_response_json(django_client, object_url, {})
         assert object_json['@id'] == object_id
         conn = BlitzGateway(client_obj=self.root)

